woodwork/type_sys/type_system.py

```diff
@@ -77,19 +80,19 @@
     (IntegerNullable, AgeNullable),
     (IntegerNullable, Integer),
     (NaturalLanguage, Address),
```

Comment:
Should it be this inference relationship? 
```suggestion
    (Unknown, Address),
```